VstsTasks/AzureDtlCreateCustomImage/new-azuredtl-customimage.json

```diff
@@ -59,16 +33,7 @@
             "properties": {
                 "author": "[parameters('author')]",
                 "description": "[parameters('description')]",
-                "osType": "[parameters('osType')]",
-                "vm": {
-                    "sourceVmId": "[parameters('sourceLabVmId')]",
-                    "linuxOsInfo":  {
-                        "linuxOsState": "[parameters('linuxOsState')]"
-                    },
-                    "windowsOsInfo": {
-                        "windowsOsState": "[parameters('windowsOsState')]"
-                    }
-                }
+                "vm": "[parameters('vmOsInfo')]"
```

Comment:
why did we remove the `osType` parameter that we were passing earlier.

Reply:
We don't use it. Only when VHD is used.